### ISSUE TITLE
Allow users to upload documents

### ISSUE THREAD

**Issue by Parth18Shah** [growupanand/ConvoForm#283]
**Is your feature request related to a problem? Please describe.** 
N/A. This feature allows organizations to create forms for more scenarios, such as job applications wherein candidates can upload their resumes and supporting documents.

**Describe the solution you'd like** 
Create a new type of field that the creators can consider adding (if needed) to their forms.

**Describe alternatives you've considered** 
N/A

**Additional context** 
I plan to research various approaches before I start the implementation process. Please let me know if you have any suggestions or inputs on how I could implement this.

**Comment by growupanand** [comment]
@Parth18Shah , I like this feature even this is in my feature roadmap. So how you are planning to do this?

**Comment by Parth18Shah** [comment]
Hello, regarding the implementation, I haven't finalized my approach yet, and as I said earlier, I would perform some research online and gather various ways to develop this. 
On basic skimming of the net, I came across Edge store (a library) which we can add to your next.js app.

**Comment by Parth18Shah** [comment]
This can be another approach that makes more sense. I am not sure about any security vulnerabilities and potential solutions to avoid spamming in this approach. Can I search about it and get back to you on Discord if that works?

**Comment by growupanand** [comment]
Yes sure, you can join discord channel [here](https://discord.gg/vYE7v3hj)
